### PR TITLE
Some small fixes

### PR DESCRIPTION
**Pull request by hillu**
I am trying to get stenographer to run on Debian and have run into a few problems so far. The certificate fix is not perfect -- domain names ought to be handled better -- but with that change done, I at least was able to establish a connection to the stenographer server.

### CODE REVIEW

File: certs/certs.go [hunk 1]
```diff
@@ -64,6 +66,7 @@ func WriteNewCerts(certFile, keyFile string, server bool) error {
 		KeyUsage:              x509.KeyUsageKeyEncipherment | x509.KeyUsageDigitalSignature | x509.KeyUsageCertSign,
 		BasicConstraintsValid: true,
 		DNSNames:              []string{"localhost"},
+		IPAddresses:           []net.IP{net.IPv4(127, 0, 0, 1)},
```
Comment (gconnell) [inline certs/certs.go]: We should probably use config.Host here?

File: config/config.go [hunk 2]
```diff
@@ -83,12 +82,12 @@ func ReadConfigFile(filename string) (*Config, error) {
 
 // Validate checks the configuration for common errors.
 func (c Config) Validate() error {
-	for _, thread := range c.Threads {
-		if _, err := os.Stat(thread.PacketsDirectory); err != nil {
```
Comment (gconnell) [inline config/config.go]: Why this change to not check for directories?

Comment (hillu) [inline config/config.go]: as explained in 180fd9b, checking for the directories prevents stenographer from creating the directories itself (`thread/thread.go`, function `makeDirIfNecessary()`).

File: certs/certs.go (hunk 1 above)
Comment (hillu) [inline certs/certs.go]: Sure. As I wrote, this is not perfect... If `config.Host` is supposed to be an IP address, the `DNSNames` is wrong, too.

Comment (gconnell) [inline certs/certs.go]: We can save this for later... someone else recently introduced config.Host, and I think there's still a couple things to hash out to make it work well, this code included.  Don't want to have that stop you from getting this in, though.
